src/jit/ssabuilder.cpp

```diff
@@ -484,8 +466,8 @@ void SsaBuilder::ComputeDominators(BasicBlock** postOrder, int count, BlkToBlkSe
     // Allocate space for constant time computation of (a DOM b?) query.
     unsigned bbArrSize = m_pCompiler->fgBBNumMax + 1; // We will use 1-based bbNums as indices into these arrays, so
                                                       // add 1.
-    m_pDomPreOrder  = jitstd::utility::allocate<int>(m_allocator, bbArrSize);
-    m_pDomPostOrder = jitstd::utility::allocate<int>(m_allocator, bbArrSize);
+    m_pDomPreOrder  = new (&m_allocator) int[bbArrSize];
```

Comment:
Shouldn't we add `Allocator* GetAllocator()` in order to avoid using `&` everywhere and make code a bit more readable?

Reply:
Hmm, maybe. Or new operator overloads that take `CompAllocator&` instead of `CompAllocator*`. But note that this comes from a commit in another PR - #14953, I only added here so I can work until that PR is merged.

Reply:
That other PR has been merge but I plan to clean up this and some other `CompAllocator` related issues in a future PR.